crates/tx5-connection/src/webrtc/libdatachannel.rs

```diff
@@ -25,27 +25,29 @@ struct Dch(CloseSend<Cmd>);
 
 impl datachannel::DataChannelHandler for Dch {
     fn on_open(&mut self) {
-        self.0.send_or_close(Cmd::DataChanOpen);
+        let _ = self.0.send_or_close(Cmd::DataChanOpen);
```

Comment:
No need to report error here because the channel can't be full when the connection is first opened?